ci/pingcap_tidb_operator_build_kind.groovy

```diff
@@ -157,6 +157,7 @@ def call(BUILD_BRANCH, CREDENTIALS_ID, CODECOV_CREDENTIALS_ID) {
 							export PATH=${WORKSPACE}/go/bin:\$PATH
 							make check-setup
 							make check
+							./hack/check-EOF.sh
```

Comment:
better to add it in https://github.com/pingcap/tidb-operator/blob/b91846337e454258f0d1c75ad2caceee33ea63de/Makefile#L137 , so that `check-EOF.sh` will be executed  in `make check`